package.json

```diff
@@ -179,6 +180,7 @@
     "jsdom": "^16.3.0",
     "lint-staged": "^10.2.11",
     "mini-css-extract-plugin": "^0.9.0",
+    "moment-locales-webpack-plugin": "^1.2.0",
```

Comment:
See above, let's revert the changes to `package.json` and `package-lock.json` as part of this issue.